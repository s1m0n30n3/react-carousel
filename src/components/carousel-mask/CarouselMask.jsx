import React from "react";

export const CarouselMask = (props) => (
  <div {...props} className="carousel-mask--base" />
);

CarouselMask.displayName = "Gallery.CarouselMask";
