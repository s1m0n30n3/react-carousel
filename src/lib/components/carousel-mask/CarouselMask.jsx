import React from "react";

import { base } from "styles/carousel-mask.module.css";

export const CarouselMask = (props) => <div {...props} className={base} />;

CarouselMask.displayName = "Gallery.CarouselMask";
