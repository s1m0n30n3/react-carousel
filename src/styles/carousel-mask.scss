.carousel-mask--base {
    width: 100%;
    height: inherit;
    overflow: hidden;
}