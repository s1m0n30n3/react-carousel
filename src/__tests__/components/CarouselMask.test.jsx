import React from "react";
import renderer from "react-test-renderer";
import { describe, expect, test } from "vitest";

import { CarouselMask } from "components/carousel-mask";

describe("CarouselMask", () => {
  test("component renders correctly without props", () => {
    const component = renderer.create(<CarouselMask />);

    const tree = component.toJSON();

    expect(tree).toMatchSnapshot();
  });
});
