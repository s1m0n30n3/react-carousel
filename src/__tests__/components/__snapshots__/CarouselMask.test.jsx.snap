// Vitest Snapshot v1

exports[`CarouselMask > component renders correctly without props 1`] = `
<div
  className="carousel-mask--base"
/>
`;
